src/lightning_app/cli/lightning_cli.py

```diff
@@ -337,6 +339,75 @@ def stop() -> None:
 _main.add_command(create)
 
 
+@_main.command("ssh")
+@click.option(
+    "--app-id",
+    "app_id",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.option(
+    "--component-name",
+    "component_name",
+    type=str,
+    default=None,
+    help="Specify which component to SSH into",
+)
+def ssh(app_id: str = None, component_name: str = None) -> None:
+    """SSH into a Lighting App."""
+    import inquirer
+
+    app_manager = _AppManager()
+    if app_id is None:
+        apps = app_manager.list_apps(phase_in=[V1LightningappInstanceState.RUNNING])
+        if len(apps) == 0:
+            raise click.ClickException("no running apps available.")
+
+        available_apps = [
+            inquirer.List(
+                "app_name",
+                message="What app to SSH into?",
+                choices=[app.name for app in apps],
+            ),
+        ]
+        app_name = inquirer.prompt(available_apps)["app_name"]
+        app_id = [app.id for app in apps if app.name == app_name][0]
+
+    try:
+        instance = app_manager.get_app(app_id=app_id)
+    except ApiException:
+        raise click.ClickException("failed fetching app instance")
```

Comment:
What is the action the user should take? Do they need to log in first?

Reply:
the user selected an app that doesn't exist- when I change the interface to accept an app_name this error message will be more descriptive.